### DISCUSSION TITLE
Linting all things in IDE - the ultimate doc

### DISCUSSION THREAD

**Discussion by zzap** [WordPress/developer-blog-content#320] [Topic Ideas]
This has been proposed as developer docs page but I think it fits better into dev blog, due to the nature of development tools and workflows to change without affecting the actual development.

This is the original issue: https://github.com/WordPress/Documentation-Issue-Tracker/issues/34

**Comment by zzap** [comment]
A [comment](https://github.com/WordPress/Documentation-Issue-Tracker/issues/34#issuecomment-2400214109) from @jrfnl:

> I wonder if there should be multiple articles for different target groups ?
> 
> How to set things up & get it running locally for contributing to WP Core
> Best practices for setting things up for a plugin/theme and how to integrate it into CI
> How to set up your local tooling environment in general (project vs global, IDEs etc)

**Comment by bph** [comment]
This topic was approved and conversation continues on the issue
- #334
